### ISSUE TITLE
HTML appearing in certain searches

### ISSUE THREAD

**Issue by capaldo** [KorySchneider/wikit#23]
When searching `wikit stroop effect` some HTML appears. For example:
```bash
 <div style="background:#f9f9f9;border:1px solid #666;float:right;width:50%;padding:1em;margin:0
 0 1em 1em;">         Naming the font color of a printed
 word is an easier and quicker task if word meaning and
 font color are congruent.
```

**Comment by KorySchneider** [comment]
Which version of wikit are you running? (`wikit --version`)

**Comment by KorySchneider** [comment]
Looks like it may just be that article is formatted strangely with added html in the summary section. I'll look into it when I have more time, feel free to open a PR in the meantime!

**Comment by KorySchneider** [comment]
Fixed in ecf112c

**Comment by KorySchneider** [comment]
Changes are now live in wikit 4.2.2, `npm i -g wikit@latest` to update
